frontend/src/components/CustomModal/CustomModal.tsx

```diff
@@ -0,0 +1,32 @@
+import Modal from "react-bootstrap/Modal";
+import { useState } from "react";
+import LoginForm from "../LoginForm";
+import SignUpForm from "../SignUpForm";
+
+interface ModalProps {
+  isOpen: boolean;
+  toggle: () => void;
+}
+
+const CustomModal = (props: ModalProps) => {
+    const [contentType, setContentType] = useState(true);
```

Comment:
This variable name is a bit confusing, maybe something like [isLogin, setIsLogin] would  be more understandable?